frontend/src/App.scss

```diff
@@ -279,3 +279,8 @@ button:not([disabled]):focus {
 .DayPickerNavigation_button svg  {
 fill: #1B1B1B;
 }
+
+/* ========== plotly overrrides ========== */
+.plotly-notifier {
+  z-index: 99999;
```

Comment:
could this have any issues with other z-order items like the drawer, loading message, or model dialog boxes?

Reply:
this notifier is only displayed for a few seconds and will therefore likely not be shown when either a modal or drawer is opened. with that said, it would be positioned above both of those elements, which I think makes sense as that's pretty typical behavior for short-lived notifications like this one

if this is ever shown at the same time the full-page loader is shown, it would be below it, but that condition is pretty unlikely I think